bitbots_hcm/src/bitbots_hcm/hcm_dsd/hcm_blackboard.py

```diff
@@ -63,7 +63,7 @@ def __init__(self):
         self.pressure_sensors_installed = rospy.get_param("hcm/pressure_sensors_installed", False)
         self.pressure_timeout_duration = rospy.get_param("hcm/pressure_timeout_duration")
         self.last_pressure_update_time = None
-        self.pressures = [0] * 8
+        self.pressures = [100] * 8
```

Comment:
Why is this changed?

Reply:
otherwise the HCM thinks its picked up (no forces on the feet) till it receives the first foot pressure message. in webots we have no pressure sensors yet, so the HCM would always stay picked up. on real robot this is also a possible race condition which could make the HCM go in the wrong state during boot

Reply:
Okay, please add a comment.